### PR TITLE
mobile web app compatability added

### PR DESCRIPTION
**Pull request by griddoz92**
i

### CODE REVIEW

**Comment by kaso17** [comment]
Is this some github bug or why do I see css in the js file (https://github.com/Jackett/Jackett/pull/1064/files)?

**Comment by griddoz92** [comment]
Fixed

**Comment by kaso17** [comment]
There still seems to be a lot of duplicated JS code  and css (starting at line 1049)
HTML code seems to have been duplicated too (There are e.g. two closing </html> tags now.
Did you make any changes except the added meta data (withing the <head> tag in the html file?

**Comment by griddoz92** [comment]
Yes I am working on the table spill over issue atm

**Comment by griddoz92** [comment]
The html should only have added meta tags and icon links

**Comment by kaso17** [comment]
ok, then please have a look at the file changed details, there are plenty of unwanted changes.

File: src/Jackett/Content/index.html [hunk 1]
```diff
@@ -54,6 +69,534 @@
             <h4>Adding a Jackett indexer in Sonarr or Radarr</h4>
             <ol>
                 <li>Go to <b>Settings > Indexers > Add > Torznab > Custom</b>.</li>
+                <li>Click on the indexers corresponding <button type="button" class="btn btn-xs btn-info">Copy Torznab Feed</button> button and paste it into the Sonarr/Radarr <b>URL</b> field.</li>
```
Comment (kaso17) [inline src/Jackett/Content/index.html]: Please remove this duplicate html code

File: src/Jackett/Content/index.html [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+<!DOCTYPE html>
 
 <html lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <meta name="mobile-web-app-capable" content="yes"/>
+    <meta name="apple-mobile-web-app-capable" content="yes"/>
     <meta charset="utf-8" />
+	<link rel="apple-touch-icon" href="../apple-touch-icon.png"/>
```
Comment (kaso17) [inline src/Jackett/Content/index.html]: seems like tabs and spaces have been merged, I suggest that you use 4 spaces instead of a tab

**Comment by griddoz92** [comment]
Sorry trying to do it on my phone at work custom.js should be fixed now

File: src/Jackett/Content/index.html (hunk 1 above)
Comment (griddoz92) [inline src/Jackett/Content/index.html]: Which line can't seem to be able to see it

**Comment by griddoz92** [comment]
Fixed it it's a bug with GitHub select all button doesn't select all so when I was pasting my code it was leaving reminents of the old code

**Comment by kaso17** [comment]
Now the tab/spaces are messed up again

**Comment by kaso17** [comment]
merged, thank you

**Comment by griddoz92** [comment]
This may also impove mobile usability 
It appears that the code is already in jackett for it
https://datatables.net/extensions/rowreorder/examples/initialisation/responsive.html

**Comment by kaso17** [comment]
The Responsive DataTables extension isn't part of jackett, you would have to add it.

**Comment by griddoz92** [comment]
Yes but the classes are already in your code

**Comment by kaso17** [comment]
that might be, but the responsive JS lib is definitely not included.

**Pull request merged**